.github/workflows/benchmark.yml

```diff
@@ -83,8 +83,7 @@ jobs:
           git remote add upstream https://github.com/facebookincubator/velox
           git fetch upstream
           git status
-          merge_base=$(git merge-base 'upstream/${{ github.base_ref }}' 'origin/${{ github.head_ref }}') || \
-            { echo "::error::Failed to find merge base"; exit 1; }
+          merge_base="d51190039648bc1b5725e0261889d19ea2e65a85"
```

Comment:
We are using a last known good merge base to validate the build works and compare results against. 
After this lands we can revert to original code to determine merge base.